source/develop/release-management/features/storage/incremental-backup.md

```diff
@@ -266,6 +267,12 @@ You need to poll the backup until the phase is `"ready"`.
 Once the backup is ready the response will include `<to_checkpoint_id>` which
 should be used as the `<from_checkpoint_id>` in the next incremental backup.
 
+In case of starting an incremental backup which includes disks that weren't part
+of the given `<from_checkpoint_uuid>`, or new disks that were attached
+to the VM after the given `<from_checkpoint_uuid>` was taken, the system will detect
+those disks and a full backup will be taken for them (`backup_mode="full"`).
+The `backup_mode` for the rest of the the disks will be `incremental` (`backup_mode="incremental"`).
```

Comment:
This text is not clear, and does not explain how this API should be used.

I think we need something like this:

    Before downloading the data, the user must check if incremental
    backup is available for a particular disk. If the system can provide
    incremental backup data for a disk, the disk backup_mode will be
    "incremetnal". in this case the user can start an image transfer,
    and preform incremental backup.

    If the system cannot provide incremental backup data for a disk, the 
    disk backup_mode will be "full". The user can start an image
    transfer and perform full backup of this disk.  The next backup for 
    this disk can be incremental.

    The system cannot provide incremental backup when:
    - adding a new disk to a VM
    - adding existing disk to a backup, which was not included in a
      previous backup specified by the `<from_checkpoint_uuid>`
    - the system find that persistent dirty bitmaps are missing on
      storage

Reply:
I've added a new section for 'Disk backup_mode' before the explanation about imageio requests